include/zephyr/sys/util_macro.h

```diff
@@ -73,6 +73,9 @@ extern "C" {
  */
 #define BIT64_MASK(n) (BIT64(n) - 1ULL)
 
+/** @brief Check if a @p x is a power of two */
+#define IS_POWER_OF_TWO(x) (((x) != 0U) && (((x) & ((x) - 1U)) == 0U))
```

Comment:
Curious: What was the reason for extracting this into a macro?

Reply:
So that it can be used at compile time